src/main/java/com/synopsys/integration/alert/common/descriptor/Descriptor.java

```diff
@@ -94,6 +93,14 @@ public void addDistributionUiConfig(final DescriptorActionApi descriptorActionAp
         return getUIConfig(context).map(uiConfig -> createMetaData(uiConfig, context));
     }
 
+    public Map<String, Boolean> getKeys(final ConfigContextEnum context) {
+        return getUIConfig(context)
+                   .map(uiConfig -> uiConfig.createFields())
+                   .orElse(Collections.emptyList())
```

Comment:
We should favor `List.of()` over `Collections.emptyList()` in Java 11.

Reply:
Dammit!